src/lib/actions/players.ts

```diff
@@ -87,13 +88,24 @@ export const searchPlayers = async (inputValue: string): Promise<IPlayer[]> => {
     const players = await Player.find({
       name: { $regex: inputValue, $options: "i" },
     }).limit(5);
```

Comment:
can't use populate?

Reply:
I guess we can't because when creating new Player we pass only teamId here. 

![image](https://github.com/miqs1992/typer2024/assets/78431445/6570d808-0267-4916-a720-d5f5e53e30bb)

We could change it but there are already a lot of players added to the production, I don't know if it would spoil anything, what do you think?